src/env.h

```diff
@@ -503,6 +503,9 @@ class Environment {
 
   inline v8::Local<v8::Object> NewInternalFieldObject();
 
+  inline void AtExit(void (*cb)(void* arg), void* arg);
+  inline void RunAtExit();
```

Comment:
`RunAtExit` was a confusing name when I first read it… I don’t have anything better in mind rn, but if you do, feel free to add that here.